modules/sugar/src/test/ts/browser/ShadowDomTest.ts

```diff
@@ -118,34 +121,84 @@ UnitTest.test('isSupported platform test', () => {
   }
 });
 
-if (ShadowDom.isSupported()) {
-  UnitTest.test('stylecontainer is shadow root for shadow root', () => {
-    withShadowElement((sr) => {
-      Assert.eq('Should be shadow root', sr, ShadowDom.getStyleContainer(sr), tElement);
-    });
+UnitTest.test('stylecontainer is shadow root for shadow root', () => {
+  withShadowElement((sr) => {
+    Assert.eq('Should be shadow root', sr, ShadowDom.getStyleContainer(sr), tElement);
   });
-}
+});
 
 UnitTest.test('stylecontainer is head for document', () => {
   Assert.eq('Should be head', Head.getHead(Document.getDocument()), ShadowDom.getStyleContainer(Document.getDocument()), tElement);
 });
 
-if (ShadowDom.isSupported()) {
-  UnitTest.test('contentcontainer is shadow root for shadow root', () => {
-    withShadowElement((sr) => {
-      Assert.eq('Should be shadow root', sr, ShadowDom.getContentContainer(sr), tElement);
-    });
+UnitTest.test('contentcontainer is shadow root for shadow root', () => {
+  withShadowElement((sr) => {
+    Assert.eq('Should be shadow root', sr, ShadowDom.getContentContainer(sr), tElement);
   });
-}
+});
 
 UnitTest.test('stylecontainer is body for document', () => {
   Assert.eq('Should be head', Body.getBody(Document.getDocument()), ShadowDom.getContentContainer(Document.getDocument()), tElement);
 });
 
-if (ShadowDom.isSupported()) {
-  UnitTest.test('getShadowHost', () => {
-    withShadowElement((sr, inner, sh) => {
-      Assert.eq('Should be shadow host', sh, ShadowDom.getShadowHost(sr), tElement);
-    });
+UnitTest.test('getShadowHost', () => {
+  withShadowElement((sr, inner, sh) => {
+    Assert.eq('Should be shadow host', sh, ShadowDom.getShadowHost(sr), tElement);
   });
+});
+
+UnitTest.test('isOpen / isClosed', () => {
+  withShadowElementInMode('open', (sr) => {
+    Assert.eq('open shadow root is open', true, ShadowDom.isOpen(sr));
+    Assert.eq('open shadow root is not closed', false, ShadowDom.isClosed(sr));
+  });
+  withShadowElementInMode('closed', (sr) => {
+    Assert.eq('closed shadow root is not open', false, ShadowDom.isOpen(sr));
+    Assert.eq('closed shadow root is closed', true, ShadowDom.isClosed(sr));
+  });
+});
+
+function checkOriginalEventTarget(mode: 'open' | 'closed', success: UnitTest.SuccessCallback, failure: UnitTest.FailureCallback) {
```

Comment:
`const checkOriginalEventTarget = ...` instead for consistency maybe?

Reply:
Fixed.